src/components/calcite-pick-list/calcite-pick-list.scss

```diff
@@ -13,6 +13,11 @@ header {
   justify-content: flex-end;
   align-items: center;
   margin-bottom: var(--calcite-app-cap-spacing-half);
+  &.sticky {
```

Comment:
this doesn't have to be a compound selector - thanks to style encapsulation. I thought it a slight advantage to indicate in the CSS which element this modifier belongs to. LMK if you think it should be otherwise.